content/docs/install/plugins.md

```diff
@@ -14,7 +14,7 @@ In order to recognize `.dvc` files as YAML in Vim, you should add:
 
 ```vim
 " DVC
-autocmd! BufNewFile,BufRead Dvcfile,*.dvc setfiletype yaml
+autocmd! BufNewFile,BufRead *.dvc setfiletype yaml
```

Comment:
Thanks @sarthakforwet 

@efiop , do you think we should still keep this for legacy projects? (see https://github.com/iterative/dvc.org/pull/1420#pullrequestreview-432694500)

Reply:
@jorgeorpinel Yes, I would keep it for now.